test/clj/web/user/test.clj

```diff
@@ -0,0 +1,16 @@
+(ns web.user.test
+  (:require 
+   [web.user :refer :all]
+   [clojure.test :refer :all]))
+
+(deftest username
+  (testing "valid-username?"
```

Comment:
Remove this testing block cuz it doesn't add anything relevant to the failure message.